substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/jfr/JFR.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020, 2021, Red Hat, Inc.
```

Comment:
I think that those files will need a different copyright header.

Reply:
What do you suggest / don't you like? I believe it's what we use elsewhere for new files.

Reply:
The gate checks the copyright header (e.g., each copyright header must mention Oracle as well). Here is one compliant example that @adinn used for a file that he added in the context of his debugging info support: https://github.com/adinn/graal/blob/2c3c8abf4c59df5492f9add84f8d5b442c56825d/substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ArrayTypeEntry.java